feature-settings/src/main/java/cm/aptoide/pt/settings/presentation/SettingsView.kt

```diff
@@ -308,22 +321,26 @@ fun SettingsScreen(
         title = "About us"
       )
       ExtraOptionText(
-        modifier = Modifier.padding(horizontal = 16.dp, vertical = 6.dp),
-        description = "Version $version",
+        modifier = Modifier.padding(horizontal = 16.dp, vertical = 12.dp),
+        description = "Version $versionName",
         // buttonTitle = "CHECK UPDATES"
       )
       ExtraOption(
-        modifier = Modifier.padding(horizontal = 16.dp, vertical = 6.dp),
+        modifier = Modifier.padding(horizontal = 16.dp, vertical = 12.dp),
```

Comment:
In case you do the refactor in this comment: https://github.com/Aptoide/aptoide-client-v8/pull/1665/files#r1265533152
You could extract a composable that creates an ExtraOption with this modifier and reuse it.